From 2e4767b2d056bd18cfa6cc6577f771d5c1dce7a1 Mon Sep 17 00:00:00 2001
From: tofylion <tofy2001@hotmail.com>
Date: Mon, 13 Mar 2023 18:27:02 +0000
Subject: [PATCH] hwdb: update 70-mouse.hwdb (#26782)

Add Logitech G502 X wired and wireless configurations with correct pid and vid.

(cherry picked from commit dede07d3d04007c70c78653a73e2bcd8616564a5)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 4084019bde..463f0ad30b 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -388,6 +388,12 @@ mouse:usb:v046dpc08b:name:Logitech G502 HERO SE:*
 mouse:usb:v046dpc08b:name:Logitech G502 HERO Gaming Mouse:*
  MOUSE_DPI=1200@1000 *2400@1000 3200@1000 6400@1000
 
+# Logitech G502 X (Wired)
+mouse:usb:v046dpc098:name:Logitech, Inc. G502 X LIGHTSPEED:*
+# Logitech G502 X (Wireless)
+mouse:usb:v046dpc547:name:Logitech USB Receiver:*
+ MOUSE_DPI=1200@1000 *2400@1000 3200@1000 6400@1000
+
 # Logitech G700 Laser Mouse (Wired)
 mouse:usb:v046dpc06b:name:Logitech G700 Laser Mouse:*
 # Logitech G700 Laser Mouse (Wireless)
